### PR TITLE
Added PR template

### PR DESCRIPTION
**Pull request by 96RadhikaJadhav**
This PR will resolve issue #29 .

### CODE REVIEW

File: .github/PR_template.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#### Issue Number
+ISSUE #
```
Comment (urvashi-code1255) [inline .github/PR_template.md:2]: @96RadhikaJadhav Please write fixes # in place of line 2.

**Review by urvashi-code1255 — COMMENTED** [review]
Excellent!! just do that one change :)

File: .github/PR_template.md (hunk 1 above)
Comment (96RadhikaJadhav) [reply .github/PR_template.md]: @urvashi-code1255  Done :) .

**Review by urvashi-code1255 — COMMENTED** [review]
@96RadhikaJadhav please write it in small letters "fixes"

**Comment by 96RadhikaJadhav** [comment]
> @96RadhikaJadhav please write it in small letters "fixes"

Done.

**Review by urvashi-code1255 — APPROVED** [review]
Thanks for incorporating changes @96RadhikaJadhav . LGTM 👍

**Pull request merged**